<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1019</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
border: 2px solid  rgb(0, 255, 0);
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="TitleBar" name="titleBar" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="body" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="videoWidget" native="true">
        <widget class="ControlBar" name="controlBar" native="true">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>540</y>
           <width>631</width>
           <height>50</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
        </widget>
        <widget class="SliderVideo" name="slider_video">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>510</y>
           <width>631</width>
           <height>22</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="Sidebar" name="sidebar" native="true">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>251</width>
           <height>491</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitleBar</class>
   <extends>QWidget</extends>
   <header>titlebar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ControlBar</class>
   <extends>QWidget</extends>
   <header>controlbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Sidebar</class>
   <extends>QWidget</extends>
   <header>sidebar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SliderVideo</class>
   <extends>QSlider</extends>
   <header>slidervideo.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
